docs/developer-docs/integrations/sns/get-sns/local-testing.md

```diff
@@ -1,74 +1,150 @@
+    3. Go to the neurons tab and create a neuron. Give it a lot of ICP (e.g. 500_000_000) and an 8 year dissolve delay.
+7. You will also need a small neuron to represent yourself, the developer. 5 ICP should suffice. You will also need to add your principal as a hotkey to this developer neuron.
+    1. Log into the NNS dapp with a different new anchor (a private browser window or a different browser can be used to log into two different anchors at the same time).
+    2. Make sure that you have at least 5 ICP in your main account; if not get more with the "Get ICP" menu entry.
+    3. Go to the neurons tab and create a neuron. Give it 5 ICP and an 8 year dissolve delay.
+    4. Make a note of your neuron ID. The rest of this guide will refer to it as `DEVELOPER_NEURON_ID`. The commands below assume you run `export DEVELOPER_NEURON_ID=<neuron id>` in your shell.
+    5. Add one of the dfx identities as a hotkey to the neuron. This dfx identity will be referred to as `developer-identity`. To get the principal of this identity, run `dfx --identity developer-identity identity get-principal`.
 
 #### 3. Ask the SNS wasm modules canister to install an SNS.
-Make a call to the SNS wasm modules canister on the local NNS 
-to request that an SNS is installed.
-To make this call, use your `dfx` identity `identityDevDfx` and
-the command as described 'here'. <!--TODO-CLI/dfx-Link: -->
-Upon receiving this call, the SNS wasm modules canister will install
-an SNS with your chosen initial parameters.
-<!--TODO-CLI/dfx-Link: once tooling is clear, make sure that here automatically
-the .yaml file is used. If this is not the case, add the information how this 
-can be ensured.-->
-
-#### 4. Add the SNS root canister as a controller to your dapp canister(s).
-To do so, use your `dfx` identity `identityDevDfx` and
-the command described 'here'.
-<!-- TODO: add this to CLI/dfx tool as need to learn SNS canisters -->
-
-#### 5. Remove all controllers other than the SNS from the dapp canister(s)
-Remove yourself, as well as any other developers,
-from the list of controllers that the dapp canister(s) have.
+Make a call to the SNS wasm modules canister on the local NNS to request that an SNS is installed.
+Installing the SNS has some preconditions:
+
+1. The SNS configuration (created in the [previous step](./preparation.md)) has to be named `sns.yml` and has to be placed in your project's root directory.
+2. Your [cycles wallet](../../../build//project-setup/cycles-wallet.md) has to be added to the whitelist of principals that are allowed to create SNSes. This is a temporary measure to make sure that no one launches an SNS before the code is ready. In the long-term, this condition will be removed. 
+3. Your cycles wallet contains enough cycles to create an SNS. Currently, this cost is 50T cycles.
```

Comment:
```suggestion
3. Your cycles wallet contains enough cycles to create an SNS. Currently, this cost is 180T cycles.
```

I think we talked about this, but not sure anymore: Maybe in Max's script this is still 50T? 
In the real SNS-W this was changed to 180T, thus if we take the master version of SNS-W I think this will also change for local testing?